Classes/commands.py

```diff
@@ -1352,13 +1352,16 @@ def get_role_members(self, role: discord.Role) -> list:
         checks.is_white_shirt(), checks.is_dev_team(), checks.is_team_lead()
     )
     @commands.command()
```

Comment:
You can add a usage="role_name" so that it's formatted a bit in the help command. Currently it looks like this:
![image](https://user-images.githubusercontent.com/33896457/119562455-dec28980-bd95-11eb-9dfd-fe986f82890c.png)